Formula/libui.rb

```diff
@@ -0,0 +1,50 @@
+class Libui < Formula
+  desc "portable GUI library for C"
+  homepage "https://github.com/andlabs/libui"
+  url "https://github.com/andlabs/libui/archive/alpha3.1.tar.gz"
+  sha256 "945ee9abacffafa93b5cf941bddfbe0bd0d4b3a8c7f8b0256779e24249f57679"
+
+  head "https://github.com/andlabs/libui/libui.git"
+
+  option "without-shared-library", "Build static library only (defaults to building dylib only)"
+  option "with-examples", "Build examples"
+  option "with-tester", "Build tester program"
```

Comment:
The test or examples are not required for the functioning of the library, I didn't want to add [possible] unnecessary build dependencies. I honestly don't even know how users would reach these executables in `libexec` normally.